addon/validators/base.js

```diff
@@ -85,25 +85,24 @@ export default Ember.Object.extend({
    * @param  {Object} context     : Context for string replacement
    * @return {String}             : The generated message
    */
-  createErrorMessage(type, options, value, context) {
+  createErrorMessage(type, options, value, context = {}) {
     options = options || {};
```

Comment:
Sounds good. Lets just leave it for now. I don't think we should change two public API's for a default arg.